CMakeLists.txt

```diff
@@ -33,14 +33,14 @@
 MESSAGE(STATUS "This is CMake ${CMAKE_VERSION}")
 MESSAGE(STATUS "")
 
-CMAKE_MINIMUM_REQUIRED(VERSION 3.1.0)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.3.0)
```

Comment:
we currently require 3.1 according to https://www.dealii.org/developer/readme.html
Do you need 3.3? If yes then we need to adjust.

Reply:
Yes. I chose to simply bump to 3.3.0. It will make some cleanup of the CMake code I plan to do easier - and version 3.3.0 is reasonably ancient (Ubuntu 16.04 LTS already shipped 3.5.0). I have adjusted the readme and the rest of the documentation to state `3.3.0`.